app/src/main/java/com/github/se/travelpouch/model/profile/Profile.kt

```diff
@@ -36,6 +37,21 @@ object ErrorProfile {
       Profile("000errorerrorerrorerrorerror", "error", "error@error.ch", null, "error", emptyList())
 }
 
+object CurrentProfile {
+  var profile = nobodyLoginProfile
+}
+
+object NobodyLoginProfile {
+  val nobodyLoginProfile =
+      Profile(
+          "0000000000000000000000000000",
```

Comment:
Yes, because I did not want to put an error profile as the initialisation state of the profile, since the error profile can be a useful test later. It is only used as an initilisation state. Also, it is better than to use Profile?, so that is why

Reply:
ok I understand but why didn't you use it?

Reply:
Yes, now it is useless. I removed it.